websites/D/dod Games/presence.ts

```diff
@@ -0,0 +1,253 @@
+const presence = new Presence({
+	clientId: "937393073539911730",
+});
+
+let isInGame = false,
+	timeStarted = Date.now();
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "logo",
+		},
+		playerName = document.querySelector(
+			"#component_top_right_loginasname_text"
+		).textContent,
+		urls = document.querySelectorAll("#topBarDownLink");
+	let openRooms = 0;
+
+	for (const url of urls) {
+		if (url.textContent.includes("&room=")) openRooms++;
+
+		// Draw It
+		if (url.textContent.includes("game=drawit&room=")) {
+			if (!isInGame) {
+				timeStarted = Date.now();
+				isInGame = true;
+			}
+
+			let round = document.querySelector("#round b").textContent;
+			round = round.replaceAll(" ", "");
+			const scores = document.querySelectorAll("#playersContainer .player");
+			for (let i = 0; i < scores.length; i++) {
+				if (scores[i].querySelector(".name").textContent === playerName) {
+					if (
+						(document.querySelector("#drawtools") as HTMLElement).style
```

Comment:
HTMLElement is an abstract class. Please apply this with the actual class this represents
```suggestion
						document.querySelector<HTMLElement>("#drawtools").style
```

Reply:
I've changed them to `HTMLDivElement` in [b47b445](https://github.com/PreMiD/Presences/pull/6342/commits/b47b44558a7a8d33935e4edac40823adf5b0a219), is that ok?